test_scripts/WidgetSupport/common.lua

```diff
@@ -102,6 +103,24 @@ function m.getOnSystemCapabilityParams(pMaxNumOfWidgetWindows)
                   characterSet = "UTF_8",
                   width = 1,
                   rows = 1
+                },
+                {
+                  name = "subtleAlertText1",
+                  characterSet = "UTF_8",
+                  width = 500,
+                  rows = 1
+                },
+                {
+                  name = "subtleAlertText2",
+                  characterSet = "UTF_8",
+                  width = 500,
+                  rows = 1
+                },
+                {
+                  name = "subtleAlertSoftButtonText",
+                  characterSet = "UTF_8",
+                  width = 500,
+                  rows = 1
```

Comment:
I would propose to make this update in specific script and not in common module which is used for all scripts for widgets